src/khoj/processor/conversation/prompts.py

```diff
@@ -122,6 +122,9 @@
     """
 You are a talented creator. Generate a detailed prompt to generate an image based on the following description. Update the query below to improve the image generation. Add additional context to the query to improve the image generation. Make sure to retain any important information from the query. Use the conversation log to inform your response.
 
+Today's Date: {current_date}
+Location: {location}
```

Comment:
How do you see today's date and user's location being used by the image generator?

Reply:
Something like `paint a picture of where i am now` might be relevant? Like depicting current city (e.g., `Shanghai`) in the current season (e.g., `winter`)? Edge case, but I can imagine it being relevant.